gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/JavaServiceDistributionExtension.java

```diff
@@ -148,6 +151,18 @@ public final void enableManifestClasspath(boolean newEnableManifestClasspath) {
         this.enableManifestClasspath.set(newEnableManifestClasspath);
     }
 
+    public final Provider<Boolean> getEnableAlwaysPreTouch() {
```

Comment:
Make this a `Property<Boolean>` then remove the getter and setter below. Gradle will use this `get...` property to make `enableAlwaysPreTouch = true` work.

Reply:
I might consider renaming this to property to `alwaysPreTouch` then having a void method `enableAlwaysPreTouch`. This means you can excavate out

```gradle
distribution {
    enableAlwaysPreTouch()
}
```

rather than `alwaysPreTouch = true`, which is a bit nicer/more in line with some of our other options in various plugins.

Reply:
I updated it to use the `enableAlwaysPreTouch()` style setter and changed the name of the property to `alwaysPreTouch`. How does it look now?

Reply:
looks good